File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        int maxfreq =0;
        int total =0;
        int Count[] = new int[101];
        for(int n : nums){
            Count[n]++;
            int freq = Count[n];
            if(freq > maxfreq){
                maxfreq = freq;
                total = freq;
            }
            else if(freq == maxfreq){ 
                total += maxfreq;
            }
        }
        return total;
        
    }
}

File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        vector<int>Count(101);
        int maxFreq =0,total=0;
        for(int &n : nums){
            Count[n]++;
            int freq = Count[n];
            if(freq>maxFreq){
                maxFreq = freq;
                total = maxFreq;
            } 
            else if( freq == maxFreq){
                total += maxFreq;
            }
        }
        return total;
        
    }
};